Resources/config/controller.xml

```diff
@@ -19,7 +19,7 @@
             <argument>%assetic.variables%</argument>
         </service>
         <service id="assetic.controller" class="%assetic.controller.class%" scope="prototype">
-            <argument type="service" id="request" />
+            <argument type="expression">service('request_stack').getMasterRequest()</argument>
```

Comment:
I'll change to `request_stack`, just the time to clone the bundle locally, squash and fix this